opennms-base-assembly/src/main/filtered/etc/elastic-flow-persistence-credentials.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Can we rename this to something more generic like  `elastic-credentials.xml`? This would allow us to reuse in other contexts, since there isn't anything specific to flows here.

Reply:
Yes, sure, I just have to place it somewhere else then.